include/alpaka/kernel/TaskKernelCpuOmp2Blocks.hpp

```diff
@@ -830,12 +830,12 @@ namespace alpaka
 
             // Body of the OpenMP parallel loop to be executed.
             // Index type is auto since we have a difference for OpenMP 2.0 and later ones
-            auto loopBody = [&](auto i) {
+            auto loopBody = [&](auto currentIndex) {
```

Comment:
Note: this is due to `i` shadowing warning in some compilers, as `i` is also used as an index of `for` loops calling this lambda. I would imagine it's a false alarm tho.